packaging/packager

```diff
@@ -56,41 +56,35 @@ if ! type zip > /dev/null 2>&1; then
     exit 1
 fi
 
-function pluck_so_files() {
+function find_so_files() {
     sed -E '/\.so$|\.so\.[0-9]+$/!d'
 }
 
 function package_libc_alpine() {
-    if grep --fixed-strings "Alpine Linux" < /etc/os-release > /dev/null; then
+    if grep -F "Alpine Linux" < /etc/os-release > /dev/null; then
         if type apk > /dev/null 2>&1; then
-            apk info --contents musl 2>/dev/null | pluck_so_files | sed 's/^/\//'
+            apk info --contents musl 2>/dev/null | find_so_files | sed 's/^/\//'
         fi
     fi
 }
 
 function package_libc_pacman() {
     if grep --extended-regexp "Arch Linux|Manjaro Linux" < /etc/os-release > /dev/null 2>&1; then
         if type pacman > /dev/null 2>&1; then
-            pacman --query --list --quiet glibc | pluck_so_files
+            pacman --query --list --quiet glibc | find_so_files
         fi
     fi
 }
 
 function package_libc_dpkg() {
     if type dpkg-query > /dev/null 2>&1; then
-        if [[ $(dpkg-query --listfiles libc6 | wc -l) -gt 0 ]]; then
-            dpkg-query --listfiles libc6 | pluck_so_files
-        fi
+        dpkg-query --listfiles libc6:$(dpkg --print-architecture) | find_so_files
```

Comment:
If it's not broken let's not try to be clever. I _really_ don't want to have churn in bash files because of some arcane syntax in some distro. We already don't have good CI for different distros, so please let's not change what is currently working.

Reply:
I understand, there's a bit of fragility here and it would be undesirable to introduce a regression.

My thoughts were that this is actually a more straightforward (and less hacky) approach in this particular case.

Admittedly, my testing wasn't totally comprehensive, but I did some runs with the packager in both a debian and fedora docker container to make sure that it properly does the job here and picks up the shared modules before submitting the PR.

To be precise, in the loop further down that iterates over the items, I added:
```bash
echo "module: $i"
```
So I could see what modules are being picked up.

Since it's natural that people will be reproducing this pattern to add support for other distros, taking the time to set a good example seems worth considering.